SCClassLibrary/Common/Collections/osx/extString_osx.sc

```diff
@@ -16,6 +16,7 @@
 					++this++"\n"
 					++"rm"+fpath.quote+"\n");
 			file.close;
+			fpath = fpath.shellQuote;
```

Comment:
move this line above what is currently line 15, and then remove `.quote` from what is currently line 17, so that all uses of `fpath` are correctly escaped. `shellQuote` escapes quotes in path names, whereas plain `quote` doesn't handle that

Reply:
thanks for the review! PR updated.